frontend/packages/knative-plugin/src/components/overview/EventSourceResources.tsx

```diff
@@ -70,27 +66,42 @@ const EventSourceResources: React.FC<EventSourceResourcesProps> = ({ obj, ownedS
       ) : (
         <span className="text-muted">{t('knative-plugin~No sink found for this resource.')}</span>
       )}
-      {podData?.pods?.length > 0 && <PodsOverview obj={obj} allPodsLink={linkUrl} />}
-      {deploymentData?.name && (
+    </>
+  );
+};
+
+export const EventSourceDeployments: React.FC<EventSourceDeploymentsProps> = ({
+  deploymentObj,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      {!_.isEmpty(deploymentObj) ? (
         <>
           <SidebarSectionHeading text={t('knative-plugin~Deployment')} />
           <ul className="list-group">
             <li className="list-group-item">
               <ResourceLink
-                kind={deploymentData.kind}
-                name={deploymentData.name}
-                namespace={namespace}
+                kind={deploymentObj.kind}
+                name={deploymentObj.metadata.name}
+                namespace={deploymentObj.metadata.namespace}
               />
             </li>
           </ul>
         </>
-      )}
-      {ownedSources?.length > 0 &&
-        ownedSources.map((source) => (
-          <EventSourceOwnedList key={source.metadata.uid} source={source} />
-        ))}
+      ) : null}
     </>
   );
 };
 
-export default EventSourceResources;
+export const EventSourceOwnedEventSources: React.FC<ventSourceOwnedEventSourcesProps> = ({
```

Comment:
```suggestion
export const EventSourceOwnedEventSources: React.FC<evntSourceOwnedEventSourcesProps> = ({
```

and this name seems confusig `EventSourceOwnedEventSources`

Reply:
Change it to `OwnedEventSources`